src/config.coffee

```diff
@@ -1190,6 +1191,13 @@ Config.addSchemaEnforcers
 isPlainObject = (value) ->
   _.isObject(value) and not _.isArray(value) and not _.isFunction(value) and not _.isString(value) and not (value instanceof Color)
 
+sortObject = (value) ->
+  return value unless _.isObject(value) and not _.isArray(value)
```

Comment:
Good point. I have addressed it in https://github.com/atom/atom/commit/fbbcdbf5185f2a08dd1705d7a11767920e662f71#diff-1dca5d0fd1e63d76f43245422a3a18efR1195